src/main/java/seedu/address/model/person/Student.java

```diff
@@ -5,6 +5,7 @@
 import java.util.Collections;
 import java.util.HashSet;
 import java.util.Objects;
+import java.util.Optional;
```

Comment:
Haha I think you can remove this? Seems like it's not used

Reply:
yea checkstyle found that